srcpkgs/cmake-bootstrap/patches/0001-We-only-want-a-half-baked-cmake.patch

```diff
@@ -1,26 +1,36 @@
+From 87468f753d9732eac35a80217638a51cd106e0eb Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Fri, 20 Nov 2020 18:55:09 -0600
+Subject: We only want a half-baked-cmake
+
+---
+ Source/cmSystemTools.cxx |  7 +----
+ bootstrap                | 67 ++--------------------------------------
+ 2 files changed, 3 insertions(+), 71 deletions(-)
+
 diff --git Source/cmSystemTools.cxx Source/cmSystemTools.cxx
-index 1e78d36..a678122 100644
+index 97440d2..747cedc 100644
 --- Source/cmSystemTools.cxx
 +++ Source/cmSystemTools.cxx
-@@ -1988,7 +1988,7 @@ void cmSystemTools::FindCMakeResources(const char* argv0)
```

Comment:
Too much noise in this patch! And you're not the author ;)

Reply:
Ok well you can update cmake-bootstrap if you like